.github/wasi_testsuite_adapter.py

```diff
@@ -0,0 +1,47 @@
+# adapter for wazero until/unless https://github.com/WebAssembly/wasi-testsuite/pull/55
```

Comment:
this adapter was the culmination of work by @achille-roussel @evacchi and @mathetake

Reply:
Oh I did nothing, and I just copy-pasta 🍝 in the PR :D